src/dynacli/bootstrap/init.py

```diff
@@ -0,0 +1,24 @@
+import os
+import shutil
+import stat
+
+
+def init(name: str, path: str) -> None:
+    """
+    Initialize the CLI entrypoint in given path
+
+    Args:
+        name (str): name of the new CLI tool
+        path (str): The path for CLI entrypoint
+
+    Return: None
+    """
+    wd = os.path.dirname(os.path.realpath(__file__))
+    cwd = path if path != "." else os.path.curdir
```

Comment:
Problematic user experience, you enforce me to type dynacli initi mycli . most of the time. The alternative would be to define **kwargs and recognise path=. In the future, you could support additional optional parameters such as description, version, root, etc.

Reply:
If you want a quick release, we may just drop the path argument and defer more advanced features to next release.

Reply:
Let's release it now, as we need to release with python 3.7 support + medium article.
Mostly this feature just created for our medium article - then when we need something serious we can improve it further,

Reply:
Switched to **kwargs and accepting path= -> if not path= provided then will go with current directory